src/psyclone/transformations.py

```diff
@@ -2318,15 +2521,19 @@ def name(self):
         '''Returns the name of this transformation as a string.'''
         return "GOLoopSwap"
 
-    def _validate(self, node_outer):  # pylint: disable=no-self-use
+    def validate(self, node_outer, options=None):
+        # pylint: disable=no-self-use
         '''Checks if the given nodes contains a valid Fortran structure
-           to allow swapping loops. This means the node must represent
-           a loop, and it must have exactly one child that is also a loop.
+        to allow swapping loops. This means the node must represent
+        a loop, and it must have exactly one child that is also a loop.
 
-           :param node_outer: A node from an AST.
-           :type node_outer: py:class:`psyclone.psyGen.Node`
-           :raises TransformationError: if the supplied node does not
-                                        allow a loop swap to be done.
+        :param node_outer: a node from an AST.
+        :type node_outer: py:class:`psyclone.psyGen.Node`
```

Comment:
It should be `Loop` rather than just `Node`.

Reply:
Done.